src/util/print_diagnostics.hpp

```diff
@@ -87,12 +87,12 @@ inline std::string power_of_10(double value) {
 
 #ifndef SLEIPNIR_DISABLE_DIAGNOSTICS
 /**
- * Prints error for too many degrees of freedom.
+ * Prints error for too few degrees of freedom.
  *
  * @param c_e The problem's equality constraints cₑ(x) evaluated at the current
  *   iterate.
  */
-inline void print_too_many_dofs_error(const Eigen::VectorXd& c_e) {
+inline void print_too_few_dofs_error(const Eigen::VectorXd& c_e) {
```

Comment:
The `#define` below also needs updating.

Reply:
I'll just deal with this in another PR.